src/components/Dashboard/TalkDashboard.vue

```diff
@@ -144,16 +142,24 @@ async function startMeeting() {
  * Scrolls the event cards wrapper in the specified direction.
  * @param {string} direction - The direction to scroll ('backward' or 'forward').
  */
-function scroll({ direction }: { direction: 'backward' | 'forward' }) {
-	const scrollDirection = direction === 'backward' ? -1 : 1
+function scrollEventCards({ direction }: { direction: 'backward' | 'forward' }) {
+	const scrollDirection = (direction === 'backward' ? -1 : 1) * (isDirectionRTL ? -1 : 1)
 	if (eventCardsWrapper.value) {
+		const ITEM_WIDTH = 300 + 8 // 300px width + 8px gap
 		let scrollAmount = 0
-		const visibleItems = Math.floor(eventCardsWrapper.value.clientWidth / (300 + 4))
+		const visibleItems = Math.floor(eventCardsWrapper.value.clientWidth / ITEM_WIDTH)
 		if (visibleItems === 0) {
-			// FIXME: mobile view, scroll by 1 item
 			scrollAmount = eventCardsWrapper.value.clientWidth * scrollDirection
+		} else {
+			scrollAmount = visibleItems * ITEM_WIDTH * scrollDirection
```

Comment:
Nitpick: it scrolls by 308+-34px, but should be scrolled to the next card border instead. Can't think of a quick solution, we can leave it for the follow-up